src/main/java/core/basesyntax/service/impl/OperationParserImpl.java

```diff
@@ -0,0 +1,40 @@
+package core.basesyntax.service.impl;
+
+import core.basesyntax.model.FruitTransaction;
+import core.basesyntax.service.OperationParser;
+import java.util.Arrays;
+import java.util.List;
+import java.util.stream.Collectors;
+
+public class OperationParserImpl implements OperationParser {
+    private static final int HEADER = 0;
```

Comment:
```suggestion
    private static final int HEADER = 1;
```

Reply:
Header is at 0 index

Reply:
I wrote it because you should use skip() method in sream. When you read from file you skip Header. That why HEADER is 1.